src/main/java/uk/gov/hmcts/reform/prl/services/StmtOfServImplService.java

```diff
@@ -243,6 +244,9 @@ private List<DynamicListElement> getRespondentsList(CaseData caseData) {
 
     public ServedApplicationDetails checkAndServeRespondentPacksPersonalService(CaseData caseData, String authorization) {
         SoaPack unServedRespondentPack = caseData.getServiceOfApplication().getUnServedRespondentPack();
+        String whoIsResponsible = SoaCitizenServingRespondentsEnum.courtAdmin
+            .toString().equalsIgnoreCase(unServedRespondentPack.getPersonalServiceBy())
+            ? PERSONAL_SERVICE_SERVED_BY_CA : PERSONAL_SERVICE_SERVED_BY_BAILIFF;
```

Comment:
For the existing cases, this might be an issue.

Reply:
after discussing with you we have found this is not an issue